Makefile

```diff
@@ -44,4 +45,5 @@ manzan-installer-v%.jar: /QOpenSys/pkgs/bin/zip appinstall.jar
 	gmake -C config BUILDVERSION="$*" install BUILDLIB=${BUILDLIB}
 	gmake -C ile BUILDVERSION="$*" BUILDLIB=${BUILDLIB}
 	gmake -C camel BUILDVERSION="$*" clean install
+	install -m 555 -o qsys service-commander-def.yaml ${INSTALL_ROOT}/opt/manzan/lib/manzan.yaml

```

Comment:
Why are we changing the file name to manzan.yaml?

Reply:
We rename it so that we can start it using:
```
sc start manzan.yaml
```
instead of:
```
sc start service-commander-def.yaml
```